pytorch_lightning/metrics/classification/auroc.py

```diff
@@ -156,12 +160,4 @@ def compute(self) -> torch.Tensor:
         """
         preds = torch.cat(self.preds, dim=0)
         target = torch.cat(self.target, dim=0)
-        return _auroc_compute(
-            preds,
-            target,
-            self.mode,
-            self.num_classes,
-            self.pos_label,
-            self.average,
-            self.max_fpr
-        )
+        return _auroc_compute(preds, target, self.mode, self.num_classes, self.pos_label, self.average, self.max_fpr,)
```

Comment:
This should be formatted by applying yapf again.
```suggestion
        return _auroc_compute(preds, target, self.mode, self.num_classes, self.pos_label, self.average, self.max_fpr,)
```

Reply:
rerun now :]